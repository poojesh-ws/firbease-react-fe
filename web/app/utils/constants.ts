export const HEADER_HEIGHT = '7rem';
export const MIN_SIDEBAR_WIDTH = '4.5rem';
export const MOBILE_DRAWER_BREAKPOINT = 450;
