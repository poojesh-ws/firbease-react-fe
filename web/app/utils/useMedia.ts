import useScreenType from 'react-screentype-hook';
import { screenBreakPoints } from '@themes/media';

export default function useMedia() {
  return useScreenType({
    mobile: screenBreakPoints.MOBILE,
    tablet: screenBreakPoints.TABLET,
    desktop: screenBreakPoints.DESKTOP,
    largeDesktop: screenBreakPoints.LARGE_DESKTOP
  });
}
