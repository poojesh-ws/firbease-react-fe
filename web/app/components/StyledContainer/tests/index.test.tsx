import React from 'react';
import { renderWithIntl } from '@app/utils/testUtils';
import StyledContainer from '..';

describe('<StyledContainer /> tests', () => {
  it('should render and match the snapshot', () => {
    const { baseElement } = renderWithIntl(<StyledContainer />);
    expect(baseElement).toMatchSnapshot();
  });
});
