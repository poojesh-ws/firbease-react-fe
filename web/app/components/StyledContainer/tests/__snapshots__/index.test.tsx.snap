// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledContainer /> tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="StyledContainer-sc-1xw91me-0 jzYaeF"
    />
  </div>
</body>
`;
